### PR TITLE
Increase max message size for dumps to support large networks

### PR DESCRIPTION
**Pull request by raman325**
See https://discord.com/channels/330944238910963714/800356888827002880/893653844813299713 and https://github.com/zwave-js/zwavejs2mqtt/issues/1769 for context. We should push a new release with this change as upstream changes in a recent version of zwave-js caused the value list to increase significantly for some devices

Will there be any issues on the HA side for large dumps? I took a look at the `HomeAssistantView` logic and I don't see any limits being applied.

### CODE REVIEW

**Comment by kpine** [comment]
You're missing the client connection.

https://github.com/home-assistant-libs/zwave-js-server-python/blob/63ccaba7fe13d7d16bfda25b3755cea802af67dc/zwave_js_server/client.py#L102-L105

HA issue is https://github.com/home-assistant/core/issues/56922.

**Comment by raman325** [comment]
I didnt think it was needed but now I am realizing this would happen during connect because of the start listening command. I'll fix it in the morning

**Comment by MartinHjelmare** [comment]
@balloob do you see any issues with larger the ws message size limit?

File: zwave_js_server/client.py [hunk 1]
```diff
@@ -102,6 +106,7 @@ async def connect(self) -> None:
             self._client = await self.aiohttp_session.ws_connect(
                 self.ws_server_url,
                 heartbeat=55,
+                max_msg_size=MAX_NETWORK_STATE_SIZE,
```
Comment (balloob) [inline zwave_js_server/client.py:109]: Why add it here? Don't we only need it in `dump.py` ?

Comment (kpine) [reply zwave_js_server/client.py:109]: No, it's on any client connection.

**Comment by balloob** [comment]
Increasing size should be fine. We trust the other side, which is a ZJS server. 

This is however not a fix, but a workaround. Should we look into doing breaking dumps in multiple pieces? (Although, ofc, then it's not a perfect snapshot)

**Comment by firstof9** [comment]
> Should we look into doing breaking dumps in multiple pieces?

Is compression an option at all?

File: zwave_js_server/client.py (hunk 1 above)
Comment (raman325) [reply zwave_js_server/client.py:109]: see https://github.com/home-assistant-libs/zwave-js-server-python/pull/303#issuecomment-932694444

**Comment by raman325** [comment]
We either need to break the dump into multiple pieces or use compression. Neither is a trivial change, and this is a faster interim solution.

**Comment by raman325** [comment]
With @balloob and @kpine 's help, this PR will add compression support to the server and will compress the `start_listening` command response as long as the client supports it,. This is a much simpler change to add to the lib so closing this one

**Pull request closed without merging**